recipes/pbzip2/test_data.html

```diff
@@ -0,0 +1,643 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2//EN">
```

Comment:
Perhaps being a bit picky - but maybe we could choose a smaller test file? This is a lot of text to throw into the history IMHO. Surely any small amount of text will compress really nicely?